app/views/podcast/show.html.erb

```diff
@@ -16,18 +16,20 @@
 
     <div class="column column-two-third">
       <div class="e-content">
-
+				
         <h3 class="listen">
-          <%= t("podcast.listen") %>
-          <% if @episode.duration.present? %>
-            <small>
-              —
-              <%= @episode.duration %>
-              <abbr title="minutes">min</abbr>
-            </small>
+          <% if !lite_mode? %>
+          	<%= t("podcast.listen") %>
+          	<% if @episode.duration.present? %>
+            	<small>
+              	—
+              	<%= @episode.duration %>
+            		<abbr title="minutes">min</abbr>
+          		</small>
+        		<% end %>
```

Comment:
This might be a tabs/spaces thing too (see below), but would you mind lining up/nesting the lines so that match up, please?

```
if
  small
    -
    ep.dur
    abbr /abbr
  /small
end
```